.github/actions/rust-prepare/action.yml

```diff
@@ -36,12 +36,6 @@ runs:
         GITLAB_SSH: ${{ inputs.gitlab_ssh }}
```

Comment:
This needs to be changed to `CI_SSH_PRIVATE_KEY: "${{ secrets.CI_SSH_PRIVATE_KEY }}"`